src/SignalRServiceExtension/Bindings/SignalRIAsyncCollectorExtensions.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Azure.SignalR.Management;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+
+namespace Microsoft.Azure.WebJobs
```

Comment:
use namespace `Microsoft.Azure.WebJobs.Extensions.SignalRService` and could remove the using reference

Reply:
[IAsyncCollector](https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs/IAsyncCollector.cs) lives in Microsoft.Azure.WebJobs. Generally we like to keep extension methods in the same namespace as the class they're extending so the extension methods always show up in intellisense.